app/helpers/locks_helper.rb

```diff
@@ -26,20 +26,22 @@ def global_locks
 
   def render_locks(resource)
     locks = (resource == :global ? global_locks : Lock.for_resource(resource))
-    render partial: '/locks/lock', collection: locks, as: :lock if locks.any?
+    render partial: '/locks/lock', collection: locks, as: :lock, locals: {show_resolve: resource} if locks.any?
   end
 
   def resource_lock_icon(resource)
-    return unless lock = Lock.for_resource(resource).first
-    text = (lock.warning? ? "#{warning_icon} Warning" : "#{lock_icon} Locked")
-    content_tag :span, text.html_safe, class: "label label-warning", title: strip_tags(lock.summary)
+    return unless locks = Lock.for_resource(resource).presence
+    text = (locks.all?(&:warning?) ? "#{warning_icon} Warning" : "#{lock_icon} Locked")
+    text += " (#{locks.count})" if locks.count > 1
```

Comment:
do we really care about the count ?

Reply:
thought it was worthwhile to distinguish when there's multiple reasons for a stage to be locked